source/Application/views/admin/tpl/manufacturer_list.tpl

```diff
@@ -76,6 +81,7 @@ window.onload = function ()
             [{if $listitem->getId() == $oxid}]
                 [{assign var="listclass" value=listitem4}]
             [{/if}]
+            <td valign="top" class="[{$listclass}][{if $listitem->oxmanufacturers__oxactive->value == 1}] active[{/if}]" height="15"><div class="listitemfloating">&nbsp</a></div></td>
```

Comment:
I found a minor copy paste bug: a missing semicolon at the end of the entity &nbsp.
As far i know this is deprecated since html5 but will still work in most browsers because it is followed by a special character.